fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/mixin/eventslifecycle/MixinEntityType.java

```diff
@@ -0,0 +1,38 @@
+package net.fabricmc.fabric.mixin.eventslifecycle;
+
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.fabricmc.fabric.api.event.entity.EntityTickCallback;
+import net.fabricmc.fabric.impl.event.EntityTypeCaller;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.util.profiler.Profiler;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(EntityType.class)
+public class MixinEntityType<T extends Entity> implements EntityTypeCaller {
+
+	@Override
+	public EntityTickCallback getEntityEvent() {
```

Comment:
public EntityTickCallback<T>; make the field an EntityTickCallback<T> too; you need to implement EntityTypeCaller<T>, not EntityTypeCaller.

Reply:
1 and 2 done, is 3 a typo?